### PR TITLE
Add aside component

### PR DESCRIPTION
**Pull request by viktoria-mahmud**

### CODE REVIEW

**Review by Cosmin-Daniel — APPROVED** [review]
Reviewed and looks fine to me.

File: source/_patterns/02-organisms/components/aside.mustache [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<aside class="aside">
```
Comment (nlisgo) [inline source/_patterns/02-organisms/components/aside.mustache:1]: Rename the component `content-aside.mustache`. I did think that `article-aside` might work but I can see the use for this on other content pages.

File: assets/sass/patterns/molecules/aside.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
+@import "../../definitions";
+
+.aside {
```
Comment (nlisgo) [inline assets/sass/patterns/molecules/aside.scss:3]: Rename to `content-aside`.

File: source/_patterns/02-organisms/components/aside.mustache (hunk 1 above)
Comment (Cosmin-Daniel) [reply source/_patterns/02-organisms/components/aside.mustache]: The component is now named 'content-aside'

File: assets/sass/patterns/molecules/aside.scss (hunk 2 above)
Comment (Cosmin-Daniel) [reply assets/sass/patterns/molecules/aside.scss]: Class and name file have been refactored to 'content-aside'

**Review by nlisgo — APPROVED** [review]

**Pull request merged**